kirovy/todo.md

```diff
@@ -0,0 +1,5 @@
+- [x] Figure out if we can put a `[cncnetmap_id]` in `.map` files and have it persist through final alert. **Yes we ar able to**
+- [ ] Figure out cncnet ladder public key
+- [ ] figure out file testing that works on linux and windows. Need to modify `MEDIA_ROOT`
+- [ ] Research which map categories to include. So far I can think of Singleplayer, skirmish, co-op.
```

Comment:
![image](https://github.com/CnCNet/cncnet-map-api/assets/17500545/31a1c155-ba07-46cd-a6a9-6a10bf33ee48)
Joined the tasklists waitlist for ya 👍